src/cljs/replumb/repl.cljs

```diff
@@ -207,22 +231,25 @@
   Read-file-fn! is an async 2-arity function with signature [file-path
   src-cb] where src-cb is itself a function (fn [source] ...) that needs
   to be called with the full source of the library (as string)."
-  [verbose? src-paths read-file-fn!]
+  [{:keys [verbose src-paths read-file-fn! cache-path search-cache-in-src-paths?]}]
```

Comment:
I had choosen explicit params here, but given that there are many more now, it is better to pass-in `opts` as a whole